LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIMission_LWCustomMission.uc

```diff
@@ -344,7 +418,24 @@ simulated function UpdateData()
 	BuildMissionPanel();
 	//RefreshNavigation();
 
-	super.UpdateData();
+	// Region Panel
+	if( LibraryPanel == none )
+	{
+		UpdateTitle('Region', GetRegion().GetMyTemplate().DisplayName, GetLabelColor(), 50);
+	}
+	
+	/*********************************** Issue #140 ***********************************
+	* THe below code is replacing the call to super.UpdateData(). The code from the parent
+	* class verion of UpdateData() is pasted here except the call to UpdateMissionSchedules()
```

Comment:
Typos, but otherwise, great. It's good to have informative comments like this that explain what's been done and why.

Reply:
Corrected the typos in the latest commit.